machines/vc.ts

```diff
@@ -302,6 +323,19 @@ export const vcMachine =
             ),
         }),
 
+        removeTamperedVcs: model.assign({
+          myVcs: (context, event) =>
+            context.myVcs.filter(vc => !context.tamperedVcs.includes(vc)),
```

Comment:
as myVcs is a list of objects. calling includes function might not work well with reference. Please use isEquals function.

and also using vc as name of parameter is confusing as its vcMetadata